javascript/ql/src/experimental/Security/CWE-348/ClientSuppliedIpUsedInSecurityCheckLib.qll

```diff
@@ -0,0 +1,78 @@
+private import javascript
+private import DataFlow::PathGraph
```

Comment:
Can you move this import to the .ql file? Then this .qll file can be used by ordinary `@kind problem` queries as well.

(the import is special because it activates some special predicates that are used by `@kind path-problem` queries)

Reply:
Thanks for pointing this out, I've learned a lot from you 👍